RELEASE.md

```diff
@@ -206,12 +206,14 @@ Once these steps are completed:
     - Updates the bpfman-operator version in it's [Makefile](https://github.com/bpfman/bpfman/blob/main/bpfman-operator/Makefile):
         - `VERSION ?= x.x.x`
     - Runs `make bundle` from the bpfman-operator directory to update the bundle version.
-    - Adds a new `examples` config directory for the release version
+    - Adds a new `examples` config directory for the release version.
+    - Modify the tag for `XDP_DISPATCHER_IMAGE` and `TC_DISPATCHER_IMAGE` in `bpfman/src/lib.rs` from `latest` to the
+      new release tag.
```

Comment:
Add note:  After release is tagged, change the version tag back back to latest and push/merge pr.